base/reflection.jl

```diff
@@ -943,16 +943,17 @@ function method_exists(f::ANY, t::ANY, world=typemax(UInt))
 end
 
 """
-    isambiguous(m1, m2, [allow_bottom_tparams=true]) -> Bool
+    isambiguous(m1, m2; ambiguous_bottom=false) -> Bool
 
 Determine whether two methods `m1` and `m2` (typically of the same
 function) are ambiguous.  This test is performed in the context of
 other methods of the same function; in isolation, `m1` and `m2` might
 be ambiguous, but if a third method resolving the ambiguity has been
 defined, this returns `false`.
 
-For parametric types, `allow_bottom_tparams` controls whether
-`Union{}` is considered a valid intersection of type parameters. For example:
+For parametric types, the `ambiguous_bottom` keyword argument controls whether
+`Union{}` is counts as an ambiguous intersection of type parameters – when `true`,
```

Comment:
"Union{} is considered an" or "Union{} counts as an"

I think the former sounds better